src/KM_Controls.pas

```diff
@@ -3447,20 +3493,36 @@ procedure TKMNumericEdit.SetValue(aValue: Integer);
 procedure TKMNumericEdit.ValidateText;
 var
   I: Integer;
+  AllowedChars: TSetOfAnsiChar;
+  OnlyMinus, IsEmpty: Boolean;
 begin
+  IsEmpty := (fText = #8); // When deleting text with Backspace last character is still in string - backspace character (#8)

```

Comment:
fixed small issue:
When deleting text with Backspace last character is still in string - backspace character (#8)